snapstack/scripts/database.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
```

Comment:
I think these should all be put in their respective snap repo's test scripts. The same should be done with the  cleanup script (snap-destroy).